keps/sig-apps/85-Graduate-PDB-to-Stable/README.md

```diff
@@ -65,6 +65,9 @@ The feature was introduced in Kubernetes 1.4 and promoted to beta in 1.5.
 It has been in beta for a long time. This document lays out the plan to promote
 it to stable.
 
+Update: PDBs was promoted to GA in 1.21. Eviction is still only in policy/v1beta1
+due to some challenges described below. Ideally Eviction should also be GA in 1.22.
```

Comment:
point-in-time statements don't age well in KEPs... can update the "Implementation History" at the bottom with PDB GA in 1.21

Reply:
Moved this to the "Implementation History" section.